js/msp.js

```diff
@@ -975,26 +993,18 @@ var MSP = {
                     offset += 2;
                     ADVANCED_TUNING.deltaMethod = data.getUint8(offset++, 1);
                     ADVANCED_TUNING.vbatPidCompensation = data.getUint8(offset++, 1);
-                    if (semver.gte(CONFIG.apiVersion, "1.20.0")) {
-                        ADVANCED_TUNING.ptermSetpointWeight = data.getUint8(offset++, 1);
-                        ADVANCED_TUNING.dtermSetpointWeight = data.getUint8(offset++, 1);
-                        ADVANCED_TUNING.toleranceBand = data.getUint8(offset++, 1);
-                        ADVANCED_TUNING.toleranceBandReduction = data.getUint8(offset++, 1);
-                        ADVANCED_TUNING.itermThrottleGain = data.getUint8(offset++, 1);
-                        ADVANCED_TUNING.pidMaxVelocity = data.getUint16(offset, 1);
-                        offset += 2;
-                        ADVANCED_TUNING.pidMaxVelocityYaw = data.getUint16(offset, 1);
-                    } else {
-                        ADVANCED_TUNING.ptermSetpointWeight = 0;
-                        ADVANCED_TUNING.dtermSetpointWeight = 0;
-                        ADVANCED_TUNING.toleranceBand = 0;
-                        ADVANCED_TUNING.toleranceBandReduction = 0;
-                        ADVANCED_TUNING.itermThrottleGain = 0;
-                        ADVANCED_TUNING.pidMaxVelocity = 0;
-                        ADVANCED_TUNING.pidMaxVelocityYaw = 0;
-                    }
                 }
                 break;
+            case MSP_codes.MSP_SPECIAL_PARAMETERS:
+                 var offset = 0;
+                 RC_tuning.rcYawRate = parseFloat((data.getUint8(offset++) / 100).toFixed(2));
+                 if (CONFIG.flightControllerIdentifier == "BTFL" && semver.gte(CONFIG.flightControllerVersion, "2.8.2")) {
```

Comment:
We don't need checks for `CONFIG.flightControllerIdentifier` any more. Non - Betaflight firmware gets a warning on connect, and only sees the CLI tab. (https://github.com/betaflight/betaflight-configurator/pull/128)